packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -233,6 +237,15 @@ function Multiselect( element ) { // eslint-disable-line max-statements
     selectionsItemLabelDom.addEventListener( 'keydown', _selectionKeyDownHandler );
   }
 
+  /**
+   * Create a unique ID based on a select's option HTML element.
+   * @param {HTMLNode} option - A option HTML element.
+   * @returns {string} A hopefully unique ID.
+   */
+  function _getOptionId( option ) {
+    return _name + '-' + option.value.trim().replace( /\s+/g, '-' ).toLowerCase();
```

Comment:
Instead of only replacing spaces should we be replacing all non-word characters? https://stackoverflow.com/a/4374890

Reply:
Kudos to that developer/regex for posting a still relevant solution 10 years later.

Updated in https://github.com/cfpb/design-system/pull/1177/commits/b888465c69841a6a8686737ef549adc24f534b11